commit deb1ab8c895cfdbee0f4f73d65bcfa727a7c836f
Author: Trent Mick <trentm@gmail.com>
Date:   2019-03-15T16:26:44-07:00 (7 months ago)
    
    TRITON-884 remove cloud analytics (ca) from Triton
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/etc/logsets.json b/etc/logsets.json
index 3086781..887abb1 100644
--- a/etc/logsets.json
+++ b/etc/logsets.json
@@ -227,7 +227,6 @@
       "amon",
       "amonredis",
       "binder",
-      "ca",
       "cloudapi",
       "cmon",
       "cnapi",
